swagger_to_sdk_config_track2.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
+  "generateOptions": {
+    "generateScript": {
+      "path": "generator automation-track2",
+      "stderr": {
+        "showInComment": "^\\[AUTOREST\\]",
+        "scriptError": "^\\[ERROR\\]",
+        "scriptWarning": "^\\[WARNING\\]"
+      }
+    }
+  },
+  "advancedOptions": {
+    "cloneDir": "src/github.com/Azure/azure-sdk-for-go",
+    "breakingChangeTracking": true
+  },
+  "initOptions": {
+    "initScript": {
+      "path": "sh ./initScriptTrack2.sh",
+      "stderr": {
+        "scriptWarning": false
+      }
+    }
+  },
+  "packageOptions": {
+    "breakingChangeLabel": "CI-BreakingChange-Go"
```

We might need to add a new label for track 2
```suggestion
    "breakingChangeLabel": "CI-BreakingChange-Go-Track2"
```